src/main/java/org/generationcp/middleware/dao/ListDataPropertyDAO.java

```diff
@@ -85,4 +85,11 @@ public GermplasmListNewColumnsInfo getPropertiesForList(final Integer listId) {
 
 		return listInfo;
 	}
+
+	public boolean hasOntologyVariableInUse(final String columnName) {
```

Comment:
Rename as `isOntologyVariableInUse`. Query can be simplified as:
`select count(1) from listdataprops where  column_name=:columnName`

Reply:
done.